urllib3/backends.py

```diff
@@ -0,0 +1,12 @@
+class Backend:
```

Comment:
if this has to be py2 and 3 compatible, do we need to specify `(object)` here to guarantee new style classes?